daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -15,6 +15,103 @@
+               # Scroll wheel
+               'set_scroll_wave', 'set_scroll_static_naga_hex_v2', 'set_scroll_spectrum_naga_hex_v2', 'set_scroll_none_naga_hex_v2', 'set_scroll_reactive_naga_hex_v2', 'set_scroll_breath_random_naga_hex_v2', 'set_scroll_breath_single_naga_hex_v2', 'set_scroll_breath_dual_naga_hex_v2',
+               # Left side
+               'set_left_wave', 'set_left_static', 'set_left_spectrum', 'set_left_none', 'set_left_reactive', 'set_left_breath_random', 'set_left_breath_single', 'set_left_breath_dual',
+               # Right side
+               'set_right_wave', 'set_right_static', 'set_right_spectrum', 'set_right_none', 'set_right_reactive', 'set_right_breath_random', 'set_right_breath_single', 'set_right_breath_dual',
+               # Can set LOGO and Scroll with custom
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1205/1205_lancehead.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": "https://assets.razerzone.com/eeimages/support/products/1205/1205_lancehead.png"
+    }
+
+    DPI_MAX = 16000
+
+    def _suspend_device(self):
+        """
+        Suspend the device
+        Get the current brightness level, store it for later and then set the brightness to 0
+        """
+        self.suspend_args.clear()
+        self.suspend_args['brightness'] = (_da_get_logo_brightness(self), _da_get_scroll_brightness(self), _get_left_brightness(self), _get_right_brightness(self))
+
+        # Todo make it context?
+        self.disable_notify = True
+        _da_set_logo_brightness(self, 0)
+        _da_set_scroll_brightness(self, 0)
+        _set_left_brightness(self, 0)
+        _set_right_brightness(self, 0)
+        self.disable_notify = False
+
+    def _resume_device(self):
+        """
+        Resume the device
+        Get the last known brightness and then set the brightness
+        """
+        logo_brightness = self.suspend_args.get('brightness', (100, 100))[0]
+        scroll_brightness = self.suspend_args.get('brightness', (100, 100))[1]
+        left_row_brightness = self.suspend_args.get('brightness', (100, 100))[2]
+        right_row_brightness = self.suspend_args.get('brightness', (100, 100))[3]
+        self.disable_notify = True
+        _da_set_logo_brightness(self, logo_brightness)
+        _da_set_scroll_brightness(self, scroll_brightness)
+        _set_left_brightness(self, left_row_brightness)
+        _set_right_brightness(self, right_row_brightness)
+        self.disable_notify = False
+
+
+class RazerLanceheadWireless(RazerLanceheadWired):
+    """
+    Class for the Razer Lancehead (Wireless)
+    """
+    USB_PID = 0x005A
+    METHODS = ['set_charge_effect', 'set_charge_colour']
```

Comment:
That looks like it overwrites the other `METHODS` from the `RazerLanceheadWired` class and as such only has `set_charge_effect` & `set_charge_colour` effects and nothing else. 

Was the wireless version actually tested?

Reply:
Sorry about that.  Good catch.  I've modified it so that it appends using the + operator.  It was either that or using .extend().  Thought this was simplest.

The wireless testing was conducted with djmulder in #540 at some point, probably before d778072b029ab0414dba3d18d535f1d9fb07d3c7.  I can request a retest and hopefully have someone report back here.